WebJobs.sln

```diff
@@ -37,6 +37,10 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "build", "build", "{3B089351
 		build\PublicKey.snk = build\PublicKey.snk
 	EndProjectSection
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "WebJobs.Extensions.EventHubs", "src\Microsoft.Azure.WebJobs.ServiceBus\EventHubs\WebJobs.Extensions.EventHubs.csproj", "{F78F19F3-0023-4A5C-823B-DA054BC51D07}"
```

Comment:
Adding a separate project for this so we can produce a separate nuget package?

Reply:
So, I think that we should have EventHubs appear like our other extensions, aka Microsoft.Azure.WebJobs.Extensions.EventHubs. Don't care much either way.

I would prefer to have consistent naming with event hubs themselves: they use Microsoft.Azure.EventHubs for their nuget

Reply:
And yes, producing a separate nuget package as there are now separate Microsoft.Azure.EventHubs and Microsoft.Azure.ServiceBus nugets